test/e2e/harness.go

```diff
@@ -65,7 +65,13 @@ func newHarness() *harness {
 			gossipPeers[publicKey.KeyForMap()] = config.NewHardCodedGossipPeer(uint16(firstRandomPort+i), "127.0.0.1")
 		}
 
-		logger := log.GetLogger().WithOutput(log.NewOutput(os.Stdout).WithFormatter(log.NewHumanReadableFormatter()))
+		os.MkdirAll(config.GetProjectSourceRootPath()+"/logs", 0755)
+
+		logger := log.GetLogger().WithTags(
+			log.String("_test", "e2e"),
+			log.String("_branch", os.Getenv("GIT_BRANCH")),
```

Comment:
these two tags can be added automatically during enrichment, where you add the caller info

Reply:
I prefer this code to be explicitly outside for now. If we see that we use it in multiple places I will move it.